alerter.py

```diff
@@ -21,5 +21,6 @@ def alert_in_celcius(farenheit):
 
 alert_in_celcius(400.5)
 alert_in_celcius(303.6)
+assert(alert_failure_count == 2)

```

Comment:
Nice and simple assert. Only thing it misses is a condition when the `alert_failure_count` is *not incremented.